test/starlark_tests/ios_sticker_pack_extension_tests.bzl

```diff
@@ -71,26 +67,6 @@ def ios_sticker_pack_extension_test_suite(name):
         tags = [name],
     )
 
-    archive_contents_test(
-        name = "{}_bitcode_test".format(name),
-        apple_bitcode = "embedded",
-        build_type = "device",
-        target_under_test = "//test/starlark_tests/targets_under_test/ios:sticker_ext",
-        binary_test_file = "$BUNDLE_ROOT/sticker_ext",
-        macho_load_commands_contain = ["segname __LLVM"],
-        tags = [name],
-    )
-
-    archive_contents_test(
-        name = "{}_strip_bitcode_test".format(name),
-        build_type = "device",
-        apple_bitcode = "none",
-        target_under_test = "//test/starlark_tests/targets_under_test/ios:sticker_ext",
-        binary_test_file = "$BUNDLE_ROOT/sticker_ext",
-        macho_load_commands_not_contain = ["segname __LLVM"],
```

Comment:
can you keep the tests, i think the only thing is you'll need to flip this to:

`macho_load_commands_contain`